helm-charts/FATE/templates/hdfs-dn.yaml

```diff
@@ -0,0 +1,138 @@
+  HDFS_CONF_dfs_webhdfs_enabled: 'true'
+  HDFS_CONF_dfs_permissions_enabled: 'false'
+  HDFS_CONF_dfs_namenode_datanode_registration_ip___hostname___check: 'false'
+  YARN_CONF_yarn_log___aggregation___enable: 'true'
+  YARN_CONF_yarn_log_server_url: "http://historyserver:8188/applicationhistory/logs/"
+  YARN_CONF_yarn_resourcemanager_recovery_enabled: 'true'
+  YARN_CONF_yarn_resourcemanager_store_class: org.apache.hadoop.yarn.server.resourcemanager.recovery.FileSystemRMStateStore
+  YARN_CONF_yarn_resourcemanager_scheduler_class: org.apache.hadoop.yarn.server.resourcemanager.scheduler.capacity.CapacityScheduler
+  YARN_CONF_yarn_scheduler_capacity_root_default_maximum___allocation___mb: '8192'
+  YARN_CONF_yarn_scheduler_capacity_root_default_maximum___allocation___vcores: '4'
+  YARN_CONF_yarn_resourcemanager_fs_state___store_uri: /rmstate
+  YARN_CONF_yarn_resourcemanager_system___metrics___publisher_enabled: 'true'
+  YARN_CONF_yarn_resourcemanager_hostname: resourcemanager
+  YARN_CONF_yarn_resourcemanager_address: resourcemanager:8032
+  YARN_CONF_yarn_resourcemanager_scheduler_address: resourcemanager:8030
+  YARN_CONF_yarn_resourcemanager_resource__tracker_address: resourcemanager:8031
+  YARN_CONF_yarn_timeline___service_enabled: 'true'
+  YARN_CONF_yarn_timeline___service_generic___application___history_enabled: 'true'
+  YARN_CONF_yarn_timeline___service_hostname: historyserver
+  YARN_CONF_mapreduce_map_output_compress: 'true'
+  YARN_CONF_mapred_map_output_compress_codec: org.apache.hadoopac.io.compress.SnappyCodec
+  YARN_CONF_yarn_nodemanager_resource_memory___mb: '16384'
+  YARN_CONF_yarn_nodemanager_resource_cpu___vcores: '8'
+  YARN_CONF_yarn_nodemanager_disk___health___checker_max___disk___utilization___per___disk___percentage: '98.5'
+  YARN_CONF_yarn_nodemanager_remote___app___log___dir: /app-logs
+  YARN_CONF_yarn_nodemanager_aux___services: mapreduce_shuffle
+  MAPRED_CONF_mapreduce_framework_name: yarn
+  MAPRED_CONF_mapred_child_java_opts: '-Xmx4096m'
+  MAPRED_CONF_mapreduce_map_memory_mb: '4096'
+  MAPRED_CONF_mapreduce_reduce_memory_mb: '8192'
+  MAPRED_CONF_mapreduce_map_java_opts: '-Xmx3072m'
+  MAPRED_CONF_mapreduce_reduce_java_opts: '-Xmx6144m'
+  MAPRED_CONF_yarn_app_mapreduce_am_env: HADOOP_MAPRED_HOME=/opt/hadoop-3.2.1/
+  MAPRED_CONF_mapreduce_map_env: HADOOP_MAPRED_HOME=/opt/hadoop-3.2.1/
+  MAPRED_CONF_mapreduce_reduce_env: HADOOP_MAPRED_HOME=/opt/hadoop-3.2.1/
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: datanode
+  labels:
+    fateMoudle: datanode
+{{ include "fate.labels" . | indent 4 }}
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      fateMoudle: datanode
+{{ include "fate.matchLabels" . | indent 6 }}
+  template:
+    metadata:
+      labels:
+        fateMoudle: datanode
+{{ include "fate.labels" . | indent 8 }}
+    spec:
+      containers:
+        - name: datanode
+          image: bde2020/hadoop-datanode:2.0.0-hadoop2.7.4-java8
```

Comment:
please update this line with federatedai/hadoop-datanode:2.0.0-hadoop2.7.4-java8